tools/make/kube.mk

```diff
@@ -59,6 +59,17 @@ kube-undeploy: kube-uninstall ## Uninstall the Envoy Gateway controller into the
 	@rm -rf $(OUTPUT_DIR)/manifests/provider
 	@rm -rf $(OUTPUT_DIR)/manifests/infra
 
+.PHONY: kube-demo
+kube-demo: ## Deploy a demo backend service, gatewayclass, gateway and httproute resource and test the configuration.
+	kubectl apply -f examples/kubernetes/httpbin.yaml
+	kubectl apply -f examples/kubernetes/gatewayclass.yaml
+	kubectl apply -f examples/kubernetes/gateway.yaml
+	kubectl apply -f examples/kubernetes/httproute.yaml
+	@echo "\nPort forward to the Envoy service using the command below"
```

Comment:
Weird that ^ is rendered as `Port forward to the Envoy service using the commane below`. A simple fix is just dropping `the command below` for both instances. Note that `command` is properly rendered in L70.

It would be nice to have a `kube-undeploy` command that deletes the manifests in reverse order.

Reply:
I fixed about typo with https://github.com/envoyproxy/gateway/pull/418/commits/60d716d799977592ff610e3365ca7d34fdc1009c

any suggestions for the clean up target - `undemo`, `demo-undeploy` or `clean-demo`

Reply:
I say `demo-undeploy` to stay consistent with `kube-undeploy`.